### PR TITLE
Components

### PR DESCRIPTION
**Pull request by lazyCode-404**
### The following was implemented: 

- Initialized the React app.
- Created a directory for components.

- Display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
- Created a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- Should have a form for adding a book (no functionality yet).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+
+ESLint
+```bash
+npx eslint .
+```
+
+## Authors
```
Comment (rloterh) [inline README.md]: - [ ] OPTIONAL
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by sole contributor.

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+
+## 📝 License
+
+This project is [MIT](https://github.com/BwalyaNonde/bookstore-app/blob/dev/LICENSE) licensed.
```
Comment (rloterh) [inline README.md]: - [ ] OPTIONAL
- MIT license link is broken, kindly endeavor to fix it.

File: README.md [hunk 3]
```diff
@@ -1 +1,72 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+- React
+
+## Live Demo (if available)
+[Live Demo Link](unavailable)
```
Comment (rloterh) [inline README.md]: - [ ] OPTIONAL
Live demo link is broken, kindly endeavor to fix it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Bwalya,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- React Router correctly implemented
- Implemented required React components


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_
<br>
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (lazyCode-404) [reply README.md]: @rloterh Thank you for approving my project. I will make the suggested changes.

**Pull request merged**